README.md

```diff
@@ -108,6 +108,10 @@ Please read [CONTRIBUTING][contributing] and [CODE_OF_CONDUCT][code_of_conduct]
 for details on our code of conduct and the process for submitting pull requests
 to us.
 
+## :handshake: Contributing to our blog
```

Comment:
It is not a contributing to our blog. It is a contributing in general. The blog is a subset of all contributions.